test/mocks.js

```diff
@@ -128,6 +130,7 @@ module.exports = {
   mockLog: mockObject(LOG_METHOD_NAMES),
   mockMailer: mockObject(MAILER_METHOD_NAMES),
   mockMetricsContext,
+  mockObject: mockObject,
```

Comment:
Is there a reason to start exporting this now? I did a search of the diff and can't see it used anywhere?

Reply:
Nope, well spotted, I think I wanted this for tests at one point and then refactored it away.